hapi/hedera-protobufs/services/consensus_update_topic.proto

```diff
@@ -87,4 +88,68 @@ message ConsensusUpdateTopicTransactionBody {
      * If unspecified, no change.
      */
     AccountID autoRenewAccount = 9;
+
+    /**
+     * Access control for update/delete of custom fees.
+     * <br/>
+     * If set, subsequent consensus_update_topic transactions signed with this
+     *  key MAY update or delete the custom fees for this topic.
+     * <p>
+     * If set to a Key containing a non-empty KeyList, the previous key and new
+     *  key MUST both sign this transaction. The previous value will be replaced.
+     * <br/>
+     * If set to a Key containing an empty KeyList, the previous key and the
+     *  admin key MUST both sign this transaction. The previous value will be
+     *  cleared.
+     * <br/>
+     * If not set, the current key SHALL NOT change.<br/>
+     * <br/>
+     * If unset in state, this field MUST NOT be set in this transaction.
+     * <br/>
+     * If not set when the topic is created, this field CANNOT be set via update.
+     * <br/>
+     */
+    Key fee_schedule_key = 10;
+
+    /**
+     * A set of keys that are allowed to submit messages to the topic without
+     * paying the topic's custom fees.
+     * <br/>
+     * If a submit transaction is signed by _any_ key included in this set,
+     *  custom fees SHALL NOT be charged for that transaction.
+     * <p>
+     * If this field is not set, the current set of keys SHALL NOT change.<br/>
+     * <br/>
+     * If this field is set, but contains an empty list, any existing fee-exempt
+     * keys SHALL be removed.
+     * <br/>
+     * fee_exempt_key_list SHALL NOT contain more than
+     *  `MAX_ENTRIES_FOR_FEE_EXEMPT_KEY_LIST` keys.
+     * <br/>
+     * fee_exempt_key_list SHALL NOT contain any duplicate keys.
+     * <br/>
+     * fee_exempt_key_list MAY contain keys for accounts that are inactive,
+     *  deleted, or non-existent.
```

Comment:
Note about wording.  The keywords SHALL and SHALL NOT refer to what the caller should expect from the network.  When describing what the network demands of the caller, it is better to use the keywords MUST and MUST NOT.
Also, I cleaned up the break usage here as well, to provide another example matching the rest of the current _specification_ text for proto files.
Note that we do not want to merge description into one long sentence; we want a short "what thing is this" sentence, then an _optional_ combination of line break and extended description.  The paragraph break that follows separates description from requirements.
```suggestion
     * A set of keys.<br/>
     * Keys in this list are permitted to submit messages to this topic without
     * paying custom fees associated with this topic.
     * <p>
     * If a submit transaction is signed by _any_ key included in this set,
     * custom fees SHALL NOT be charged for that transaction.<br/>
     * If this field is not set, the current set of keys SHALL NOT change.<br/>
     * If this field is set, but contains an empty list, any existing fee-exempt
     * keys SHALL be removed.<br/>
     * A fee_exempt_key_list MUST NOT contain more than
     * `MAX_ENTRIES_FOR_FEE_EXEMPT_KEY_LIST` keys.<br/>
     * A fee_exempt_key_list MUST NOT contain any duplicate keys.<br/>
     * A fee_exempt_key_list MAY contain keys for accounts that are inactive,
     * deleted, or non-existent.<br/>
```

Reply:
Fixed